reducer/src/main/java/com/graphicsfuzz/reducer/ReductionDriver.java

```diff
@@ -132,8 +158,23 @@ public ReductionDriver(ReducerContext context,
       corePasses.add(pass);
       cleanupPasses.add(pass);
     }
-    this.passManager = new SystematicReductionPassManager(initialPasses, corePasses, cleanupPasses);
+    return new SystematicReductionPassManager(initialPasses, corePasses, cleanupPasses);
+  }
+
+  private static IReductionPassManager getLiteralsToUniformsPassManager(
+      ReducerContext context,
+      boolean verbose) {
+    final List<IReductionPass> initialPasses = new ArrayList<>();
+    final List<IReductionPass> corePasses = new ArrayList<>();
```

Comment:
I would get rid of 'initialPasses' and 'corePasses' and use Collections.emptyList() where you use them.  If you don't want to do that then how about initializing them to Collections.emptyList() here?